_data/hazemessamm/hazemessamm.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Hazem Essam
+institution: Arab Academy for Science and Technology
+profile_pic: profilepic.jpg
+quote: During COVID-19, I looked at the bright side of the quarantine that I have more time to learn new stuff.
```

Comment:
The quote does not meet the character length requirements (100 characters max, current character count: 104).